library/Booster/Pattern/Rewrite.hs

```diff
@@ -483,58 +487,61 @@ showPattern title pat = hang 4 $ vsep [title, pretty pat.term]
-|                             |                           v              v                    v                 v          |       v             |  |
-|                             |                                                                                            |                     |  |
-|                             |                         P' == _|_    P' /= _|_           /\ PS == _|_      PS simplify to  |   PS simplify to  --+  |
-|                             |                                                                                   []       |      single P'         |
-|              +--------------+-------------+               |           | |                   |                            |                        |
-|              |              |             |               |           | |                   |                   |        |                        |
-|              v              v             v               |           | |                   |                   |        +-------+                |
-|                                                           |           | |                   |                   |                v                |
-|          Does not     Simplified      Simplifies          |  +--------+-+-------------------+                   |                                 |
-|          simplify      already                            |  |        | |                                       |          PS simplify to         |
-|                                                           |  |        | |                                       |                PS'              |
-|              |              |             |               |  |        | |                                       |                                 |
-|              |              |             |               v  v        | |                                       |                 |               |
-|              |              |             |                           | +-----------------+                     |                 v               |
-|              |              |             |        Return vacuous P   |                   |                     |                                 |
-|              |              |             |                           |                   |                     |         Return branching        |
+|              +----------------------------+             v              v                    v                 v          |       v             |  |
+|              |              |             |                                                                              |                     |  |
+|              v              v             v           P' == _+_    P' /= _+_           /\ PS == _+_      PS simplify to  |   PS simplify to  +-+  |
```

Comment:
the software might have changed these : `P' == _+_`, they should be `P' == _|_`

Reply:
good catch, thanks!